### PR TITLE
#157356491 ft add coveralls badge 

### PR DESCRIPTION
**Pull request by mutoromike**
**What does this PR do?**

Adds a Coveralls badge and provides builds information from Travis CI. 

**Build Information**
screen shot 2018-05-16 at 07 47 03

**Added badge**
screen shot 2018-05-16 at 07 53 53

**What are the relevant pivotal tracker stories?**

Add coveralls badge to the repo.
ID: #157356491

### CODE REVIEW

**Review by Kalela — APPROVED** [review]
Great job

**Review by meshnesh — APPROVED** [review]
good work

**Review by lakivisi-zz — CHANGES_REQUESTED** [review]
put coveralls beside the travis badge horizontally

**Review by lakivisi-zz — CHANGES_REQUESTED** [review]
put coveralls beside the Travis badge horizontally

Is it integrated to all builds on GitHub,

File: .travis.yml [hunk 1]
```diff
@@ -66,3 +67,6 @@ script:
 
 notifications:
     slack: andela:jCFp66ZuJynangG2t3KVF3fV
+
+after_success:
```
Comment (betsybeth) [inline .travis.yml]: LGTM

**Review by FelixWambiri — APPROVED** [review]
Good Implementation.

**Comment by mutoromike** [comment]
ReStructured Text formatting does not allow me to do that.
New format:
screen shot 2018-05-17 at 13 03 20

**Comment by mutoromike** [comment]
All builds will be picked up once merged to develop.

**Review by betsybeth — APPROVED** [review]
LGTM

**Review by frankip — APPROVED** [review]
👍 
LGTM

**Comment by mutoromike** [comment]
screen shot 2018-05-17 at 14 45 09

**Review by lakivisi-zz — APPROVED** [review]

**Pull request merged**